### PR TITLE
add event section, open browser on build

### PR DESCRIPTION
**Pull request by matopher**
okc_indie_hackers

This adds a section to list upcoming events on our homepage. It also adds a `--open` flag to the dev command to open the browser automatically.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "dev": "parcel index.html",
+    "dev": "parcel index.html --open",
```
Comment (ryanlabouve) [inline package.json]: Strictly speaking, should be in separate PR. Not a big deal here.

Comment (matopher) [reply package.json]: 👍🏻 Makes sense.

**Comment by matopher** [comment]
@ryanlabouve Added a screenshot of the new section to the description.

**Comment by matopher** [comment]
@ryanlabouve Looks like I'm **not** authorized to merge, but let me know if there's anything else I can do. 👍🏻
